pytorch_lightning/utilities/meta.py

```diff
@@ -312,12 +312,20 @@ def search(mod: ModuleType) -> List[ModuleType]:
             setattr(mod, subclass.__name__, _MetaClass)
 
 
+def mock_isinstance(A, B, isinstance=None):
```

Comment:
Could you add a brief docstring why we need this?

Reply:
Done !